### ISSUE TITLE
Exercise-2 baseline solution

### ISSUE THREAD

**Issue by bkopf** [Kudos5/tdt4258#6]
We have to implement a baseline (polling) solution in order to measure and compare energy efficiency.
As I understand it we're supposed to continuously poll the counter value in addition to the button states? In other words, *no interrupts* should be enabled for this version.

**Comment by Kudos5** [comment]
Indeed. The code for polling and interrupt is mostly written. It just needs to be separated into different files.
